package-lock.json

```diff
@@ -3016,7 +3016,8 @@
       "version": "1.3.0",
       "resolved": "https://registry.npmjs.org/growly/-/growly-1.3.0.tgz",
       "integrity": "sha1-8QdIy+dq+WS3yWyTxrzCivEgwIE=",
-      "dev": true
+      "dev": true,
+      "optional": true
```

Comment:
These changes to `package-lock.json` were the result of my running `npm install` after cloning the repo.